examples/amp-rate-limited.amp.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>Rate Limited Input Examples in AMP</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+</head>
+<body>
+    <h4>On Change Slider</h4>
+    <p>This slider's value should update on mouse up, after the user finishes dragging the slider.</p>
+    <form
+      id="change-slider"
+      target="_blank">
```

Comment:
This form tag has a validation error! If you open open this test page on localhost and add`#development=1` to the end of the URL, the validator will run and you'll be able to see anything that makes the document invalid. Here the `form` tag needs to have an `action` attribute. Give a look to some of the examples from the `examples/forms.amp.html` page.

Reply:
Thanks for the `#development=1` protip! Found a few other validation issues as well with the default value of `amp-bind`. One odd thing, the validator doesn't seem to be recognizing any of my amp scripts as valid amp scripts. For example: 

It complains `Only AMP runtime 'script' tags are allowed, and only in the document head.` on this line: 
```
    <script async src="https://cdn.ampproject.org/v0.js"></script>
```

and `The attribute 'src' in tag 'amp-form extension .js script' is set to the invalid value '/dist/v0/amp-form-0.1.max.js'` on this line: 
```
    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
```

This behavior is consistent for every example I click on locally. Is this behavior intended or am I doing something wrong? I'm running: `gulp build && gulp`.

Reply:
Ah that is because our `app.js` server which is run by `gulp serve` rewrites the URLs to point to the local builds instead of the CDN. You can safely ignore those  validation errors for these files. You can change the rewriting behavior by navigating to `localhost:8000/serve_mode=` and appending one of `default` (uncompiled), `compiled` or `cdn`.

And you should just be able to just run `gulp` since the default task runs `gulp build`.